src/main/java/com/example/springBoot_hibernate_ferrocarriles/service/EquipoDeTraccionService.java

```diff
@@ -0,0 +1,45 @@
+package com.example.springBoot_hibernate_ferrocarriles.service;
+
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.example.springBoot_hibernate_ferrocarriles.model.EquipoDeTraccion;
+import com.example.springBoot_hibernate_ferrocarriles.repository.EquipoDeTraccionRepository;
+
+@Service
+public class EquipoDeTraccionService {
+	@Autowired
+	private  EquipoDeTraccionRepository equipoDeTraccionRepository;
+	
+	public EquipoDeTraccionService() {
+		super();
+	}
+
+	public EquipoDeTraccionService(EquipoDeTraccionRepository equipoDeTraccionRepository) {
+		super();
+		this.equipoDeTraccionRepository = equipoDeTraccionRepository;
+	}
+	
+	public List<EquipoDeTraccion> getAllEquipoDeTraccion(){
+		List<EquipoDeTraccion> equiposDeTraccion = new ArrayList<EquipoDeTraccion>();
+		equipoDeTraccionRepository.findAll().forEach(equipoDeTraccion1->equiposDeTraccion.add(equipoDeTraccion1));
+		return equiposDeTraccion;		
+	}
+	
+	public EquipoDeTraccion getEquipoDeTraccionById(Long idEquipoDeTraccion){
```

Comment:
Another example of logging:
```java
public EquipoDeTraccion getEquipoDeTraccionById(Long idEquipoDeTraccion){
log.info("Retrieving traction equipment by ID: {}", idEquipoDeTraccion);
...
}
```

Reply:
There a lot of loggers out there, but a very easy one is to use a library called Lombock, that among other interesting things provides a log by just annotating the class with `@Slf4j`
Download lombock [here](https://projectlombok.org/download)
Guide to set up your IDE: [this blog](https://www.baeldung.com/lombok-ide)
Anything that you can find about Java in the above blog(baeldung) is interesting and worthy knowing